app/views/content_items/world_location_news_article.html.erb

```diff
@@ -19,34 +19,38 @@
 <%= render 'govuk_publishing_components/components/notice', @content_item.withdrawal_notice_component  %>
 
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds content-bottom-margin">
-    <div class="responsive-bottom-margin">
-      <%= render 'components/figure',
-          src: @content_item.image["url"],
-          alt: @content_item.image["alt_text"],
-          credit: @content_item.image["credit"],
-          caption: @content_item.image["caption"] if @content_item.image %>
-
-      <%= render 'govuk_publishing_components/components/govspeak', {
-        direction: page_text_direction,
-      } do %>
+  <div class="govuk-grid-column-two-thirds">
```

Comment:
This whole section/general pattern seems to repeat (broadly) on a lot of pages. Would it be possible to consolidate it into a partial?